website/addons/dropbox/templates/dropbox_user_settings.mako

```diff
@@ -1,4 +1,4 @@
-## Template for the "Dropbox" section in the "Configure Add-ons" panel
+<!-- Template for the "Dropbox" section in the "Configure Add-ons" panel-->
```

Comment:
Mako comments are fine here. We use mako comments for stuff that doesn't need to be in the actual HTML page, e.g. docstrings for templates.